.github/workflows/client.yml

```diff
@@ -99,7 +97,17 @@ jobs:
           username: _json_key
           password: ${{ secrets.KEEP_TEST_GCR_JSON_KEY }}
 
+      - name: Build Docker Runtime Image
```

Comment:
Why are we splitting the docker building step? The main difference is pushing, can't we do this with `push: ${{ github.event_name == 'workflow_dispatch' }}` as we did before?

Reply:
> Execute the step reading the env variables even when the workflow was triggered by PR or push. As environment specify ropsten (reading of the values would not have much consequences)

Maybe we could use `local` as the default environment, we would add a `local.env` file that will hold some dummy variables?

Reply:
On the second thought, let's not overengineer it. The current solution is fine. Thanks for clarifying everything in your first comment!

I would just move `Build Docker Runtime Image` above `Login to Google Container Registry`, so it keeps the stages more sperated.

Reply:
Order of the steps changed in c0b590d5982c38392a7822d0eaa43783ab4c2682.